packages/deploy-tooling/src/base/deploy.ts

```diff
@@ -328,7 +418,7 @@ async function tryDeploy(
             logger.info('Deployment Successful.');
         }
     } catch (error) {
-        await handleError(TryCommands.Deploy, error, service, config, logger, archive);
+        await handleError(TryCommands.Deploy, error, provider, config, logger, archive);
```

Comment:
Do we need to rollback the TR creation if we get here. Or maybe the TR should not be created until after deployment? We dont seem to check or use the returned TR...

Reply:
The newly created TR is assigned to the `config.app.transport` and used in the deployment. But it does raise a good point, if we have successfully created a TR but the deployment to the UI5 Abap Repo failed (due to auth issues for example), we shouldn't create a second TR. 
I've updated to set `config.createTransport` to false when a TR is created succesfully

Reply:
If a 401 error is returned then the TR wouldnt have been created either?